src/main/kotlin/app/hashers/CodeLongevity.kt

```diff
@@ -253,7 +262,7 @@ class CodeLongevity(private val serverRepo: Repo,
             Files.createDirectories(Paths.get(storageDir))
         }
         val oStream = ObjectOutputStream(FileOutputStream(storagePath))
```

Comment:
This call should be encapsulated with try-catch, cuz can produce exceptions.

Reply:
same, what do you want to see under a catch block?